src/css/custom.css

```diff
@@ -225,4 +230,12 @@ table, tr, td, th {
 
 .hard {
   color: rgb(255 45 85);
+}
+
+/*  for emoji's */
```

Comment:
remove this comment. below css is self-explanatory already.

Reply:
okay